FFCDemoPaymentService/Properties/launchSettings.json

```diff
@@ -12,15 +12,15 @@
     "IIS Express": {
       "commandName": "IISExpress",
       "launchBrowser": true,
-      "launchUrl": "weatherforecast",
+      "launchUrl": "healthz",
```

Comment:
Why was this changed?

Reply:
because there is no endpoint weatherforecast in the dotnet code. It was a bug in the original code